pitch.md

```diff
@@ -7,14 +7,28 @@ _Who are you helping, with your language? What is that person's
 experience like now? How would their experience be better if you could help 
 them?_
 
+My language is for people who want to draw tableau trees, more specifically for students in CS 81.
```

Comment:
"What do you like about their idea?" --> I think this is a great DSL because it's very domain specific, and seems like it can be integrated well with languages and programs the user is already familiar with.